libraries/AP_Camera/examples/send_camera_information.py

```diff
@@ -0,0 +1,98 @@
+import threading
+import time
+from pymavlink import mavutil
+import argparse
+
+class CameraTrackingScript:
+    def __init__(self, ip, port, sysid, compid):
+        self.ip = ip
+        self.port = port
+        self.sysid = sysid
+        self.compid = compid
+        self.connection = None
+
+    def connect_to_mavlink(self):
+        self.connection = mavutil.mavlink_connection(f'udp:{self.ip}:{self.port}', source_system=self.sysid)
+        print("Searching Vehicle")
+        while not self.connection.probably_vehicle_heartbeat(self.connection.wait_heartbeat()):
+            print(".", end="")
+        print("\nFound Vehicle")
+        self.connection.wait_heartbeat()
+        print("Heartbeat received from system (system %u component %u)" % (self.connection.target_system, self.connection.target_component))
+        self.connection.mav.heartbeat_send(
+            type=mavutil.mavlink.MAV_TYPE_ONBOARD_CONTROLLER,
+            autopilot=mavutil.mavlink.MAV_AUTOPILOT_INVALID,
+            base_mode=0,
+            custom_mode=0,
+            system_status=mavutil.mavlink.MAV_STATE_UNINIT,
+            mavlink_version=3
+        )
+
+    def send_camera_information(self):
+        self.connection.mav.camera_information_send(
+            int(time.time() * 1000) & 0xFFFFFFFF,          # time_boot_ms
+            b"CameraVendor123" + b'\0' * (32 - len("CameraVendor123")),  # vendor_name
+            b"CameraModel123" + b'\0' * (32 - len("CameraModel123")),    # model_name
+            (1 << 24) | (0 << 16) | (0 << 8) | 1,          # firmware_version
+            float('nan'),                                  # focal_length
+            float('nan'),                                  # sensor_size_h
+            float('nan'),                                  # sensor_size_v
+            640,                                           # resolution_h
```

Comment:
ideally we should pull these resolutions from the rtsp stream that we're using.  Not a blocker though